vendor/assets/stylesheets/dvl/core/forms.scss

```diff
@@ -176,73 +172,111 @@ select.datetime {
   display: inline;
 }
 
-input[type="checkbox"] {
-  -webkit-appearance: none !important;
-  -moz-appearance: none !important;
-  appearance: none !important;
-  border: 1px solid $gray;
-  border-radius: $radius;
-  background-color: #fff;
-  padding: 7px;
+// Checkbox and radio inputs
+
+.control_styled {
+  width: 18px;
+  height: 18px;
   display: inline-block;
+  border: 1px solid #999;
+  margin-right: 0.5rem;
   position: relative;
-  cursor: pointer;
-  &:active,
-  &:checked:active {
-    box-shadow: 0 1px 2px rgba(0,0,0,0.05), inset 0px 1px 3px rgba(0,0,0,0.1);
+  top: 4px;
+  user-select: none;
+  &:after {
+    position: absolute;
+    display: inline-block;
+    opacity: 0;
   }
-  &:checked:after {
-    font-family: 'FontAwesome';
-    content: '\f00c';
-    font-size: 12px;
+}
+
+label.control {
+  display: block;
+  font-weight: normal;
+  font-size: 1rem;
+  position: relative;
+  line-height: 1.5rem;
+  input {
+    opacity: 0;
     position: absolute;
-    top: 1px;
-    left: 1px;
-    color: $linkColor;
+    top: 0;
+    left: 0;
+    &.readonly,
+    &.disabled {
+      + .control_styled {
+        opacity: 0.6;
+        background: #f5f5f5;
+      }
+    }
+    &:focus + .control_styled {
+      border-color: $darkBlue;
+    }
+    &:hover + .control_styled {
+      background: #f9f9f9;
+    }
+    &:active + .control_styled {
+      background: #f1f1f1;
+    }
+    &:checked + .control_styled:after {
+      opacity: 1;
+      transform: scale(1) translateZ(0);
+    }
+  }
+}
+
+
+input[type="checkbox"] {
+  + .control_styled {
+    border-radius: 2px;
+    position: relative;
+    &:after {
+      position: absolute;
+      font-family: 'FontAwesome';
+      content: "\f00c";
+      color: $darkBlue;
+      line-height: 16px;
+      width: 16px;
+      text-align: center;
+      display: inline-block;
+      opacity: 0;
+      transform: scale(0.5) translateZ(0);
+      transition: opacity 120ms $easeInOutBack, transform 170ms $easeInOutBack;
+    }
   }
   &.readonly,
   &.disabled {
-    opacity: 0.5;
+    + .control_styled:after {
+      color: $darkBlue;
```

Comment:
Name of variable `darkBlue` should be written in all lowercase letters with hyphens instead of underscores